packages/fast-components-react-base/src/select/select.schema.ts

```diff
@@ -50,6 +50,61 @@ export default {
             title: "Labelled by",
             type: "string",
         },
+        menuFlyoutConfig: {
```

Comment:
actually what you could do here is add this object as a separate export and just merge it in here, then you wont be in danger of this falling out of sync, since this looks like a copy from the viewport positioner schema

Reply:
Flyout props now extends viewport positioner props (omit "anchor")

Reply:
Sure, I meant though that you can do that here in the schema as well. These are all `.ts` files, therefore you can import them just as you would any other javascript object. Then your schema could look like:

```
import FlyoutSchema from "./path/to/schema.ts";

{
    // select schema props
    menuFlyoutConfig: FlyoutSchema
}
```

Reply:
ok

Reply:
Doesn't like it when I bootstrap that.
ReferenceError: FlyoutSchema is not defined
    at Object.<anonymous> (D:\repos\fast-dna\packages\fast-components-react-base\src\select\.tmp\select.schema.ts.js:53:27)
    at Module._compile (internal/modules/cjs/loader.js:774:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:785:10)
    at Module.load (internal/modules/cjs/loader.js:641:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Module.require (internal/modules/cjs/loader.js:681:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at D:\repos\fast-dna\build\convert-schemas.js:40:32
    at Array.forEach (<anonymous>)
    at copySchemaFiles (D:\repos\fast-dna\build\convert-schemas.js:21:47)

Reply:
Oh, you want to leave off the ".ts" from the path probably, looks like its trying to add ".js"

Reply:
We discovered an issue that needs to be fixed before we can do this:
https://github.com/microsoft/fast-dna/issues/2302